### PR TITLE
[release-0.49] Bump kubevirtci

### PR DESCRIPTION
**Pull request by fossedihelm**
**What this PR does / why we need it**:
Cherry-pick 
[https://github.com/kubevirt/kubevirt/pull/7198](https://github.com/kubevirt/kubevirt/pull/7198)
[https://github.com/kubevirt/kubevirt/pull/7165](https://github.com/kubevirt/kubevirt/pull/7165)

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: automation/test.sh [hunk 1]
```diff
@@ -54,6 +54,8 @@ elif [[ $TARGET =~ sig-compute-realtime ]]; then
   export KUBEVIRT_PROVIDER=${TARGET/-sig-compute-realtime/}
   export KUBEVIRT_HUGEPAGES_2M=512
   export KUBEVIRT_REALTIME_SCHEDULER=true
+elif [[ $TARGET =~ sig-compute-migrations ]]; then
+  export KUBEVIRT_PROVIDER=${TARGET/-sig-compute-migrations/}
 elif [[ $TARGET =~ sig-compute ]]; then
```
[the remark this answers is not in the document]
Comment (jean-edouard) [reply automation/test.sh:59]: This PR is just a cherry-pick. Looks like the main branch hasn't bumped to what you're referring to yet.

[the remark this answers is not in the document]
Comment (jean-edouard) [reply automation/test.sh:59]: @dhiller I believe you're referring to https://github.com/kubevirt/kubevirt/pull/7167
That PR is not cherry-picked by this one, and it doesn't have to be. It doesn't fix any bug, it just relocates a functional test.

**Review by dhiller — APPROVED** [review]

**Pull request merged**